data/human/free worlds start.txt

```diff
@@ -331,65 +322,101 @@ mission "FW Syndicate Diplomacy 1B"
 	landing
 	name "Tarazed Diplomacy"
 	description "Transport Alondo Gruyere to <planet> to negotiate with Tarazed Corporation."
-	source Hephaestus
-	destination Wayfarer
-	to offer
-		has "FW Syndicate Diplomacy 1: done"
+	source "Hephaestus"
+	destination "Wayfarer"
 	clearance "Alondo has a brief chat with the spaceport controller and manages to get you permission to land."
 		attributes "dirt belt"
 	passengers 1
+	to offer
+		has "FW Syndicate Diplomacy 1: done"
 	on fail
 		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
 	
-	npc
-		personality heroic
-		government Republic
-		system Sol
-		fleet "Small Republic" 3
-	
 	on offer
 		log "Visited the Syndicate and worked out a deal where they promised to stay neutral. They are expecting to profit from selling weapons to both sides in this war. Also discovered that Katya is in the Republic's custody and is accused of terrorist activity."
 		log "People" "Katya Reynolds" `Katya was on Clink when the Navy invaded the Kornephoros system, and was taken captive by them and imprisoned as a war criminal.`
+		log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public affairs, and he seemed to help get the Free Worlds a better deal with the Syndicate than they might have otherwise received. He and Alondo seem to have a history together, although it is one that neither will discuss in detail.`
```

Comment:
```suggestion
		log "People" "Samuel Remington" `Samuel Remington is the Syndicate's head of public relations, and he seemed to help get the Free Worlds a better deal with the Syndicate than they might have otherwise received. He and Alondo seem to have a history together, although it is one that neither will discuss in detail.`
```

Reply:
Did you request this as a reminder, or are you calling for discussion here?

Reply:
It's a mirror to a thing in my PR, and also probably a nudge for me to see that this is where stuff is for my reference.